assets/scripts/oppia-player-0.0.4.js

```diff
@@ -0,0 +1,374 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
```

Comment:
Just a question -- how does this differ from 0.0.3? If it is exactly the same, then you can probably just update the 0.0.3 test file instead of creating an entirely new version.